packages/dx-react-grid/docs/guides/filtering.md

```diff
@@ -9,35 +9,39 @@ The following plugins implement filtering features:
 - [LocalFiltering](../reference/local-filtering.md) - performs local data filtering
 - [TableFilterRow](../reference/table-filter-row.md) - renders a filter row
 
-Note that [plugin order](../README.md#plugin-order) is important.
+Note that [plugin order](./plugin-overview.md#plugin-order) is important.
```

Comment:
Hm, it seems is broken some time ago across all guides... Can you prepare a separate PR with a fix?

Reply:
Ok.